oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RecordRequests.jsx

```diff
@@ -13,7 +13,12 @@ import {
 } from "@tanstack/react-query";
 import { httpVnd } from "@js/oarepo_ui";
 import { OverridableContext, overrideStore } from "react-overridable";
-const overriddenComponents = overrideStore.getAll();
+import RequestsOverrides from "@js/oarepo_requests_common/RequestsOverrides";
+
```

Comment:
Put actual overrides here (common pattern in Invenio), if possible. Name RequestsOverrides is confusing (Camel-casing suggests it is a React component)